src/ResourceManager/Resources/Commands.Resources/Models.ResourceGroups/ResourceClient.cs

```diff
@@ -378,13 +377,13 @@ public virtual List<GenericResourceExtended> FilterResources(FilterResourcesOpti
             return resources;
         }
 
-        public ProviderOperationsMetadata GetProviderOperationsMetadata(string providerNamespace)
+        public Management.Resources.Models.ProviderOperationsMetadata GetProviderOperationsMetadata(string providerNamespace)
```

Comment:
Because in the new 'Microsoft.Azure.Management.Authorization' package, There has 'ProviderOperationsMetadata' function, too. ProviderOperationsMetadata' is an ambiguous reference between 'Microsoft.Azure.Management.Authorization.Models' and 'Microsoft.Azure.Management.Resources.Models' And in the old code there reference the models function from 'Management.Resources.Models' , So I add this.

'

'

'